src/components/MainNavBar/MainNavBar.tsx

```diff
@@ -81,11 +107,15 @@ const MainNavBar = () => {
                 </ListItem>
                 <ListItem className="activity-button">
                   <button
-                    onClick={() => setDOpen(true)}
+                    onClick={() =>
+                      setDOpen(!tgtValueFromStorage ? true : false)
+                    }
                     data-testid="UMLS-connect-button"
                   >
                     <div className={TGT ? "active" : "inactive"} />
```

Comment:
This line should also check for local storage like the next line in order to apply the green bubble for styling

Reply:
@mcmcphillips Thank you so much Matt! That's a good catch! I just pushed in the change.